upt_macports/upt_macports.py

```diff
@@ -15,11 +15,12 @@ def __init__(self):
     def create_package(self, upt_pkg, output):
         self.upt_pkg = upt_pkg
         self.logger.info(f'Hello, creating the package')
+        portfile = self._render_makefile_template()
```

Comment:
"portfile" can refer to:

- The path to the Portfile ("/home/cyril/macports/py-foo/Portfile")
- The file object itself
- The content of the file

So maybe we could choose another name, like "portfile_content". Same for all other occurrences in this file.